license-scanner/scanner.ts

```diff
@@ -143,6 +142,9 @@ export const scan = async function (options: ScanOptions): Promise<ScanResult> {
       logger.debug(`Excluding file ${file.path}`);
       continue toNextFile;
     }
+    if (fileExtensions.length > 0 && !fileExtensions.some((ext) => file.path.endsWith(ext))) {
```

Comment:
`fileExtensions.length > 0` check seems excessive

Reply:
True, it doesn't change anything. Removed.

Reply:
Actually it is required :D

If there are no extensions specified, the following condition will be always true and we will skip every file, which is not what we want.

Reply:
ah, dammit, sorry 😅